views/install.sh.erb

```diff
@@ -169,6 +169,14 @@ elif test "x$os" = "xAIX"; then
   platform="aix"
   platform_version="`uname -v`.`uname -r`"
   machine="powerpc"
+elif test -f "/etc/os-release"; then
+  . /etc/os-release
```

Comment:
are there any var inside of /etc/os-release that might collide with vars inside of install.sh?